src/server/job_recov_db.c

```diff
@@ -106,6 +107,14 @@
 
 #ifndef PBS_MOM
 extern pbs_db_conn_t	*svr_db_conn;
+#ifndef WIN32
```

Comment:
Would be tidier if you move the conditional compilation directives inside the body of `print_backtrace()`